rosbag2_transport/src/rosbag2_transport/rosbag2_transport_python.cpp

```diff
@@ -122,13 +127,15 @@ rosbag2_transport_record(PyObject * Py_UNUSED(self), PyObject * args, PyObject *
   bool include_hidden_topics = false;
   if (
     !PyArg_ParseTupleAndKeywords(
-      args, kwargs, "ssssss|bbKKKKObO", const_cast<char **>(kwlist),
+      args, kwargs, "ssssssKK|bbKKKKObO", const_cast<char **>(kwlist),
```

Comment:
why not optional?

Reply:
Ah, that was just an oversight, I'll fix that.